quickstart/nodejs/src/index.ts

```diff
@@ -0,0 +1,93 @@
+import { DefaultAzureCredential } from '@azure/identity';
+import { EntraIdCredentialsProviderFactory, REDIS_SCOPE_DEFAULT } from '@redis/entraid';
+import { createCluster } from '@redis/client';
+import * as net from 'node:net';
+
+const resourceEndpoint = process.env.AZURE_MANAGED_REDIS_HOST_NAME!;
+if (!resourceEndpoint) {
+    console.error('AZURE_MANAGED_REDIS_HOST_NAME is not set. It should look like: `cache-name.region-name.redis.azure.net`. Find the endpoint in the Azure portal. Do not include the port.');
+    process.exit(1);
+}
+// Azure Managed Redis default port
+const resourcePort = 10000
+
+let client;
+let endpointUrl = `rediss://${resourceEndpoint}:${resourcePort}`;

```

Comment:
This looks like a URL, but isn't actually a URL so we shouldn't call it that. Can we consolidate `endpointUrl` and `resourceEndpoint` into a single `resourceEndpoint` variable?